kanselarij-vlaanderen/kaleidos-frontend
# KAS-1294: Printable Agenda

![image](https://user-images.githubusercontent.com/1874332/75563306-711c9600-5a4a-11ea-9a37-8b870b8d6413.png)

**WHAT DOES THIS ADD?**
This adjust the behaviour of the components when you print the webpage in your browser viewing an Agenda. There are several new components **alongside** the existing agenda items list. So this basically double-renders the agenda list at the moment, but we're going to look into performance optimisations in the future if they are needed.

**HOW TO TEST THIS**
You're going to need an agenda with at least the following items to fully test the new printed layout:

* 7 or more agendaitems
* At least some announcements
* Some agendaitems should have some documents
* Some agendaitems should have an assigned mandatee
* All agendaitems should have short and long titles

If you have all these, just simply use CTRL/CMD+P or use the new action in the actions menu:
![image](https://user-images.githubusercontent.com/1874332/75564023-a4abf000-5a4b-11ea-9a78-d462a0c47941.png)

⚠️ The print will not really work on Firefox very well, because that issue was not on master when this issue was branched. But I've tested it there with the fix locally, and it behaves like Chrome. 🔥 🦊 👍 Great success!

Check JIRA issue to find the actual PDF they sent us as a sample for this view.
